config.json

```diff
@@ -3,7 +3,7 @@
   "name": "LNPoS",
   "short_description": "LNPoS lightning point of sale",
   "tile": "/lnpos/static/image/lnpos.png",
-  "min_lnbits_version": "1.0.0",
+  "min_lnbits_version": "1.1.0",
```

Comment:
Next one is 1.2.0

Reply:
i think 1.10 is fine relevant change for here was already in